libs/db/boltdb.go

```diff
@@ -190,31 +190,31 @@ func (bdb *boltDBBatch) WriteSync() {
 
 func (bdb *boltDBBatch) Close() {}
 
-// WARNING: Any concurrent writes (Set, SetSync) will block until the Iterator
-// is closed.
+// WARNING: Any concurrent writes (Set, SetSync) or reads will block until the
```

Comment:
Is a `Delete` not also a write op? I think, I'd remove `(Set, SetSync)` from the comment, or, add all write ops and read ops, too.